articles/quickstart/native/xamarin/01-login.md

```diff
@@ -98,7 +98,7 @@ public class MainActivity : Activity
 }
 ```
 
-Replace `YOUR_ANDROID_PACKAGE_NAME` in the code sample above with the actual Package Name for your application, such as `com.mycompany.myapplication`. Also ensure that all the text for the `DataScheme`, `DataHost`, and `DataPathPrefix` is in lower case.
+Replace `YOUR_ANDROID_PACKAGE_NAME` in the code sample above with the actual Package Name for your application, such as `com.mycompany.myapplication`. Also ensure that all the text for the `DataScheme`, `DataHost`, and `DataPathPrefix` is in lower case. Set `LaunchMode = LaunchMode.SingleTask`, otherwise the system will create a new instance of the activity.
```

Comment:
Set `LaunchMode = LaunchMode.SingleTask`, otherwise the system will create a new instance of the activity every time the Callback URL gets called.